src/ceph-daemon

```diff
@@ -1320,7 +1320,7 @@ def list_daemons():
                     if not host_version:
                         out, err, code = call(['ceph', '-v'])
                         if not code and out.startswith('ceph version '):
-                            host_version = out.split(' ')[3]
+                            host_version = out.split(' ')[2].split('-')[0]
```

Comment:
i don't think we want the split() part.. if the version string is ``ceph version 15.0.0-6967-ge326dfbc612 (sha1here)`` then we want ``15.0.0-6967-ge326dfbc612``.